dpc/src/network/testnet1.rs

```diff
@@ -120,6 +121,7 @@ impl Network for Testnet1 {
     const RECORD_SIZE_IN_BYTES: usize = 280;
     const RECORD_CIPHERTEXT_SIZE_IN_BYTES: usize = 320;
```

Comment:
Note to self: This might be wrong. Ensure there is a test that checks this value before merging the PR.